conn_test.go

```diff
@@ -298,6 +298,45 @@ func TestConn_List(t *testing.T) {
 	m.AssertExpectations(t)
 }
 
+func TestConn_ListHeader(t *testing.T) {
+	assert2 := assert.New(t)
```

Comment:
please use `assert := assert.New(t)`

Reply:
Would you like all instances in the file changed? Every other use is `assert2 := assert.New(t)`.